### ISSUE TITLE
Have to run `grow run` every time I update CSS

### ISSUE THREAD

**Issue by marchbox** [grow/grow#955]
Exists in both 0.8.2 and 0.8.3.

To reproduce:
1. `grow run`, site can load properly
2. Update some CSS, reload the site
3. Grow throws error:

    ```
    404: /css/composite/global-bb628161156c3c37aeb248bd32fce88c.min.css
    Traceback (most recent call last):
      File "grow/server/main.py", line 118, in __call__
      File "grow/server/main.py", line 228, in wsgi_app
      File "grow/server/main.py", line 166, in dispatch_request
    RouteNotFoundError: '/css/composite/global-bb628161156c3c37aeb248bd32fce88c.min.css was not found in routes.'

    ```

Deleting routescache.json won't work. Have to rerun `grow run` to work.

**Comment by Zoramite** [comment]
@marchbox what version of gulp are you using?

**Comment by marchbox** [comment]
@Zoramite 4.0.2

**Comment by Zoramite** [comment]
Ok, from the testing that I have done, it appears to be related to this issue: https://github.com/gulpjs/gulp/issues/2193

Specifically, the mtime of the file doesn't get changed when gulp outputs the file. Grow doesn't see that the file has changed so it doesn't update the routes.

So, this appears to be something that will need to be updated in the gulpfile. Something similar to this might work as a work-around: https://github.com/gulp-community/gulp-less/issues/301

I have another idea that would get rid of the need to watch for the files to change for fingerprinted files, but not sure what other effects it would have at the moment. I'll keep this issue open until I have the other change working.

**Comment by marchbox** [comment]
Yup, this works for now:
https://github.com/gulp-community/gulp-less/issues/301#issuecomment-420780708

Thanks, Randy.

**Comment by Zoramite** [comment]
Fixed by #966
